README.md

```diff
@@ -1,3 +1,59 @@
 # Kinto React Boilerplate
 
 A sample [Kinto](https://github.com/mozilla-services/kinto.js) & [React](http://facebook.github.io/react/) project boilerplate.
+
+## Scope
+
+The main purpose of this boilerplate is to speed up the conception of a Kinto JavaScript application, with basic setup for unit and functional testing. It comes with:
+
+* [React + ES6](https://facebook.github.io/react/blog/2015/01/27/react-v0.13.0-beta-1.html) classes, using [Babel](https://babeljs.io);
```

Comment:
None of the semi-colons `;` in this list are necessary and should be replaced with periods `.` (or nothing at all).